.travis.yml

```diff
@@ -29,10 +29,23 @@ jdk:
   - openjdk-ea
 
 script:
-  - mvn -V --no-transfer-progress
   # japicmp requires a package
-  - mvn -V jar:jar japicmp:cmp -P japicmp --no-transfer-progress
+  - |
+    if [ "$TRAVIS_JDK_VERSION" = "openjdk8" ];
+    then
+    mvn -V;
+    mvn -V jar:jar japicmp:cmp -P japicmp;
+    else
+    mvn -V --no-transfer-progress;
+    mvn -V jar:jar japicmp:cmp -P japicmp --no-transfer-progress;
+    fi;
 
 after_success:
   # jacoco will run in the main script. Include the profile to submit to coveralls.
-  - mvn -V jacoco:report coveralls:report -Ptravis-jacoco --no-transfer-progress
+  - |
+    if [ "$TRAVIS_JDK_VERSION" = "openjdk8" ];
+    then
+    mvn -V jacoco:report coveralls:report -Ptravis-jacoco;
+    else
+    mvn -V jacoco:report coveralls:report -Ptravis-jacoco --no-transfer-progress;
+    fi;

```

Comment:
@kinow 
Hi.
> Couldn't we just drop --no-transfer-progress?
I personally have no experience in using `--no-transfer-progress`, thus I cannot answer this question.
Sorry for that.
I suggest if you want to delete it, find people who add it and get the reason why they add it first.
I just assume, it is added with a reason...

Reply:
No problem @XenoAmess . I had to google that too. I also didn't know about that option, but looking at the docs I think that only reduces the log lines.

Will chase the commit history, and ping the mailing list if necessary. I noticed [lang] has that option too, though it doesn't seem to differentiate JVM version? https://github.com/apache/commons-lang/blob/master/.github/workflows/maven.yml

Reply:
@kinow 
Seems:
only jdk8 on ppc64le(which is not used in commons-lang's ci) have that trouble.
And jdk8 on AMD64 works fine.

Reply:
@kinow 
I'm trying to change the if condition from jdk-version to os-name.
If it can work, then the problem is from ppc64le but not jdk8.
(A little curious : why we need a test environment on ppc64le? Is there some historical reason?)

Reply:
@kinow
I cannot find how to get ARCH variable from travis-ci's official docs.
So I'm just, trying every String that seems possible lol.
But seems:
deleting `--no-transfer-progress` will not get the build from success to fail.

So I'm personally +1 on deleting `--no-transfer-progress`